samples/WinRTSample/WinRTSample.vsimporter/WinRTSample-WinStore10/WinRTSample.vcxproj

```diff
@@ -201,5 +215,7 @@
   </ItemGroup>
   <ItemGroup />
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
-
+  <ItemGroup>
+    <None Include="project.json" />
+  </ItemGroup>
```

Comment:
Vsimporter should have grouped this in the item group with everything else. Don't know what went on here.